pkg/test/runner/runner.go

```diff
@@ -140,15 +140,23 @@ func (r *Runner) runPipeline() error {
 	}
 
 	// run function
+	var fnErr error
 	command := run.GetMain()
 	kptArgs := []string{r.cmd, "run", tmpPkgPath}
 	for i := 0; i < r.testCase.Config.RunTimes; i++ {
 		command.SetArgs(kptArgs)
-		err = command.Execute()
+		fnErr = command.Execute()
+	}
+
+	// run formatter
+	command.SetArgs([]string{"cfg", "fmt", tmpPkgPath})
```

Comment:
Is there a reason why `kpt pipeline run` and `kpt fn run` shouldn't automatically format the output. 

@droot 
@mortent

Reply:
thanks for bringing it up, I will talk to @droot more about this when he comes back tomorrow